models/plan.py

```diff
@@ -132,3 +133,28 @@ def is_closed(self) -> bool:
         Get the read only status of the plan.
         """
         return self.__is_closed
+
+    def plan_statistics_function(self):
```

Comment:
Since its a method of plan, the word is redundant
Similarly, since this is a function, the word function is redundant.
Hence, i suggest shortening this method to `def statistics(self)`

Reply:
Fixed!